src/components/form/FieldTypeFractional.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  disabled: boolean,
+  displayError: boolean,
+  input: Object,
+  isDirty: boolean,
+  placeholder?: string,
+  setRefForField?: Function,
+}
+
+const NUMERATOR = 0;
+const DENOMINATOR = 1;
+const SPLITTER = ' / ';
+const FieldTypeFractional = ({
+  disabled = false,
+  displayError = false,
+  input: {name, onChange, value},
+  isDirty = false,
+  setRefForField,
+}: Props): React$Node => {
+
+  const handleSetReference = (element: any) => {
+    if(setRefForField) {
+      setRefForField(element);
+    }
+  };
+
+  const changeHandler = (value: string, fieldType: number): void => {
+    const values = value.split(SPLITTER);
+    values[fieldType] = value;
+    onChange(`${values[NUMERATOR] || ''}${SPLITTER}${values[DENOMINATOR] || ''}`);
+  };
+
+  const parseValue = (fieldType: number): string => {
+    if (!value) {
+      return '';
+    }
+    const values = value.split(SPLITTER);
+    return values[fieldType];
+  };
+
+  return (
+    <div className={classNames('form-field__fractional', {'has-error': displayError}, {'is-dirty': isDirty})}>
+      {/* numerator / fin: osoittaja */}
+      <input
+        ref={handleSetReference}
+        className="form-field__fractional-input"
+        id={`${name}-numerator`}
+        value={parseValue(NUMERATOR)}
+        onChange={(e) => changeHandler(e.target.value, NUMERATOR)}
+        disabled={disabled}
+        type="text"
```

Comment:
Should be of type number, same with the other field.

Reply:
done